### DISCUSSION TITLE
If I have two different dyndns providers, does that mean I need to run two different instances of godns?

### DISCUSSION THREAD

**Discussion by anoduck** [TimothyYe/godns#213] [Q&A] [answered]
If I have two different dyndns providers, does that mean I need to run two different instances of godns? One for each service provider?

Example:

site1.com uses cloudflare for dyndns.
site2.com and site3.com use he.net for dyndns. 

So I would have run one instance for cloudflare and another for he.net.

**Comment by TimothyYe** [comment, marked as answer]
For the current implementation, yes.

**Reply by anoduck** [reply]
Good to know.

**Comment by binbla** [comment]
> For the current implementation, yes.

有一个实例支持多个provider的更新计划吗？

**Reply by anoduck** [reply]
否。目前，每个 DNS 提供商只有一个实例。 多提供商实例尚不可用。
这有意义吗？ 使用谷歌翻译。

**Reply by TimothyYe** [reply]
想了解一下多个provider的使用场景是什么？

Want to know what is the use case of multiple providers?

**Reply by anoduck** [reply]
@TimothyYe Go ahead and shoot, Timothy. I have wanted to configure this for a while.

**Reply by TimothyYe** [reply]
OK, I will put it in the next release plan.

**Reply by TimothyYe** [reply]
@anoduck The multiple provider feature has just been released.

**Reply by anoduck** [reply]
@TimothyYe You know something, you are dang good. I am totally doing this.
